CHANGELOG.md

```diff
@@ -30,6 +30,8 @@ see if you need to change any of these:
 - Added a new function `unit_is_parseable` in PEcAn.utils to replace `udunits2::ud.is.parseable`.
   (#3002; @nanu1605)
 - Initial LDNDC model coupling
+- `PEcAn.settings::read.settings()` now strips comments so HTML style comments (e.g. `<!-- a comment -->`) are now allowed in pecan.xml files
+- `PEcAn.logger::setLevel()` now invisibly returns the previously set logger level
```

Comment:
Can you add this to base/logger/NEWS.md as well? Sorry for the multiple scattered changelogs...

Reply:
Should I bump the version too?

Reply:
Yes, thanks!